source/developers-guide/backend-components/basics/index.md

```diff
@@ -69,75 +75,70 @@ The backend contains some Shopware specific components, which are based on diffe
 We will start by writing our own small backend application with standard components. First we have to create a plugin which provides us with some basic features. They are:
 
 * A menu item in the backend
-* Your own backend controller
 * Your own Doctrine model
+* Your own backend controller
 
 To speed things up, our plugin is expected to create a new product list to cover all sections of our backend components. Since the doctrine models are an essential part of the backend development and modules, this tutorial will rebuild and explain some parts of the `Shopware\Models\Article` namespace.
 
-### The Plugin Bootstrap - `Bootstrap.php`
-First, you have to create a directory called `SwagProduct` with a new file called `Bootstrap.php` in it. The following features will be implemented in the bootstrap class:
+### The Plugin File
+First, you have to create a directory in `..\custom\plugins` called `SwagProduct` with a new file called `SwagProduct.php` in it. The following features will be implemented in the plugin file class:
```

Comment:
didn't you changed the name to SwagProductBasic?

Reply:
D